CDTDatastore/query/CDTQIndexManager.h

```diff
@@ -75,47 +77,49 @@ typedef NS_ENUM(NSInteger, CDTQQueryError) {
 /**
  Constructs a new CDTQIndexManager which indexes documents in `datastore`
  */
-+ (CDTQIndexManager *)managerUsingDatastore:(CDTDatastore *)datastore
-                                      error:(NSError *__autoreleasing *)error;
++ (nullable CDTQIndexManager *)
+managerUsingDatastore:(CDTDatastore *)datastore
+                error:(NSError *__autoreleasing __nullable *__nullable)error;
```

Comment:
From the [Nullability and Objective-C blog](https://developer.apple.com/swift/blog/?id=25), I don't think it's necessary to have the nullability annotations for `NSError **` in an audited region:

> The particular type `NSError **` is so often used to return errors via method parameters that it is always assumed to be a nullable pointer to a nullable `NSError` reference.